mteb/tasks/Classification/heb/Hebrew_Sentiment_Analysis.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification  # type: ignore
+from mteb.abstasks.TaskMetadata import TaskMetadata  # type: ignore
+
+
+class Hebrew_Sentiment_Analysis(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="Hebrew_Sentiment_Analysis",
+        dataset={
+            "path": "hebrew_sentiment",
+            "revision": "952c9525954c1dac50d5f95945eb5585bb6464e7",
+        },
+        description="HebrewSentiment is a data set consists of 12,804 user comments to posts on the official Facebook page of Israel’s president, Mr. Reuven Rivlin. In October 2015, we used the open software application Netvizz (Rieder, 2013) to scrape all the comments to all of the president’s posts in the period of June – August 2014, the first three months of Rivlin’s presidency.2 While the president’s posts aimed at reconciling tensions and called for tolerance and empathy, the sentiment expressed in the comments to the president’s posts was polarized between citizens who warmly thanked the president, and citizens that fiercely critiqued his policy.",
+        reference="https://huggingface.co/datasets/hebrew_sentiment",
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
```

Comment:
There are test sets, so better use them
```suggestion
        eval_splits=["test"],
```